README.md

```diff
@@ -529,6 +529,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Drishtant Dubey](https://github.com/DRISHTANT-DUBEY)
   - [Durvesh Kumar Pal](https://github.com/DurveshKumarPal)
   - [Dušan Tanasić](https://github.com/Duk4)
+  - [Darshan Jain](https://github.com/darshanjain-entrepreneur)
```

Comment:
Please add your name after Darsh Gupta, in line 477.
Let us know if you need any help with that! 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable